bayesian/utility/v2/ca_response_builder.py

```diff
@@ -18,11 +18,51 @@
 from urllib.parse import quote
 import logging
 from bayesian.utility.db_gateway import GraphAnalyses
-from bayesian.utils import version_info_tuple, convert_version_to_proper_semantic
+from bayesian.utils import version_info_tuple, convert_version_to_proper_semantic, \
+    server_create_analysis
+from typing import Dict, List, Tuple, Union, Set
+import re
+from collections import namedtuple
+from abc import ABC
+from flask import g
```

Comment:
What is the use of this import?

Reply:
`g` is used here https://github.com/fabric8-analytics/fabric8-analytics-server/blob/f5c71b2d2e3a07ce3b0f44c053be6f1646d9db89/bayesian/utility/v2/ca_response_builder.py#L46 

I think its some type of UUID

Reply:
backbone code which calls unknown packages ingestion doesn't seem to be using that parameter. Can you check?
https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/src/v2/stack_aggregator.py#L385

Reply:
Actually it looks like only `componentApiflow` need this .
https://github.com/fabric8-analytics/fabric8-analytics-server/blob/736b821da6dd08dcf2b0b8a4c2bc6c4667778e18/bayesian/utils.py#L99 

AFAIK From Backbone, this flow is not triggered.